sycl/source/detail/platform_impl.hpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 #pragma once
+#include "CL/sycl/device.hpp"
```

Comment:
```suggestion
```
Is device declaration used in this header?
Please, either remove this include or forward declaration of `device` class.